oidc_example/op2/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.6
+COPY requirements.txt /op2/
+WORKDIR /op2/
+EXPOSE 8040
+RUN pip install -r requirements.txt
+COPY . /op2/
+ENTRYPOINT ./server.py -p 8040 -t -d config_simple.py
```

Comment:
This Dockerfile, seems to still not-be-working. Can you please remove it from the PR so we can merge the stuff that's accurate? Open new PR with accompanying README update when you got it fixed! :)

Reply:
Fixed!